### ISSUE TITLE
Use cosmodal

### ISSUE THREAD

**Issue by jonator** [osmosis-labs/osmosis-frontend#650]
https://github.com/chainapsis/cosmodal

We should move our wallet connect logic to an imported cosmodal to serve as an example for others integrating with Keplr.

**Comment by jonator** [comment]
[Tweet](https://twitter.com/noahsaso/status/1540454169001152512?s=21&t=4r92Oxr3aFxcRH4pRuR8-w)

Waiting on improvements from a daodao dev to be merged into cosmodal

**Comment by jonator** [comment]
We'll likely use cosmos kit to manage wallets
